app/mongo_db_connection.py

```diff
@@ -819,7 +819,7 @@ def authorize(document_id, viewers, process_id, item_type):
     return
 
 
-def finalize_item(item_id, state, item_type):
+def finalize_item(item_id, state, item_type, message):
```

Comment:
Confirm all the item_types that get affected especially if we have `clone` also. Clone is a document copy. Ask Uche and Couzy for Clarity on this

Reply:
Done